modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js

```diff
@@ -141,13 +141,13 @@ function downloadAgent() {
         values[this.name] = $(this).val();
     });
 
-    var connectedCupRegisterURL = "/connectedcup_mgt/connectedcup/cup/register?" +
-                                                        "name=" + payload.name + "&owner=" + payload.owner;
-
     var payload = {};
     payload.name = $inputs[0].value;
     payload.owner = $inputs[1].value;
```

Comment:
How bracket notation could be more reliable? It is all about the language semantics. Using dot notation is permitted in JS as long as attribute name is a valid identifier.
http://stackoverflow.com/a/4968448/1391687